packages/paste-core/components/disclosure/src/utils.ts

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import {GenericThemeShape} from '@twilio-paste/theme';
+import {BoxStyleProps} from '@twilio-paste/box';
+import {HeadingProps} from '@twilio-paste/heading';
+import {ChevronDisclosureExpandedIconProps} from '@twilio-paste/icons/esm/ChevronDisclosureExpandedIcon';
+
+/**
+ * Returns the icon size based on the heading variant
+ * @param {HeadingProps['variant']} variant
+ * @returns {ChevronDisclosureExpandedIconProps} sizeIcon token for each heading variant
+ */
+export const getIconSize = (variant: HeadingProps['variant']): ChevronDisclosureExpandedIconProps['size'] => {
```

Comment:
Can convert this switch to a mapping:
```
{
  'heading10': 'sizeIcon90',
  ...
}
```
And if key not found default to sizeIcon20

Reply:
Good suggestion, and more performant.